GeosCore/dust_mod.F90

```diff
@@ -181,8 +180,9 @@ SUBROUTINE CHEMDUST( Input_Opt,  State_Chm, State_Diag, &
     ENDIF
 
 #ifdef APM
-
-    ! Set pointer to species vector to access concentrations
+    ! Initialize GEOS-Chem tracer array [kg] from Chemistry State
```

Comment:
Most of the diffs are due to the update to `Spc(N)%Conc(I,J,L)` in the newer code.  These may cause merge conflicts.